src/content/en/updates/2016/08/removing-document-write.md

```diff
@@ -72,9 +72,10 @@ Chrome will not execute the `<script>` elements injected via `document.write()`
    rendering of the main page.
 3. The script in the `document.write()` is parser-blocking. Scripts with the '[`async`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async)' 
    or '[`defer`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-defer)' attributes will still execute.
-4. The script is not already in the browser HTTP cache. Scripts in the cache 
+4. The script is not hosted on the same site. In other words, Chrome will not intervene for scripts with a matching eTLD+1 (e.g. a script hosted on js.example.org inserted on www.example.org).
```

Comment:
In future, please wrap text at 80 characters, I'll fix this time to get it out with this mornings push.